ls1intum/Artemis
# `Programming exercises`: Add more information to grading table

### Checklist
#### General


#### Client

### Motivation and Context


#5649 
Make the creation of test weights simpler, easier and less error-prone by providing more detail.

### Description

This PR adds some small things to make the grading table better:
- Each test now displays the amount of points it will grant for passing in the grading table.
- In the right column the total sum of all weights is now shown.
- The headers of the columns of the test case table are now also available in German.
- Since inactive test cases are already greyed out, the column for activity is not necessary and it did not provide any opportunity to sort the rows anyway. Therefore it is removed to save space.

### Steps for Testing

Prerequisites:
- 1 Instructor
- 2 Students
- 1 Programming Exercise with Complaints enabled

1. Log in to Artemis
2. Create a programming exercise
3. Go to the grading view
4. Change any values and check the shown values.

### Review Progress





#### Code Review
#### Manual Tests

### Test Coverage


| Class/File | Branch | Line |
|------------|-------:|-----:|
| programming-exercise-configure-grading.component.ts | 72% | 84% | 


### Screenshots


<img width="1671" alt="Bildschirmfoto 2022-09-26 um 01 58 04" src="https://user-images.githubusercontent.com/38322605/192171500-223009fb-fde8-47b4-a6bb-02b95ddfaffb.png">
